blurb/blurb.py

```diff
@@ -749,6 +749,10 @@ def find_editor():
         editor = os.environ.get(var)
         if editor is not None:
             return editor
+    editor = subprocess.check_output(
+        ["git", "config", "--get", "core.editor"]).strip()
+    if editor:
+        return editor
```

Comment:
I think it will be useful if we can add some instructions on how people can setup their favorite editor for blurb, or at least some explanation on how blurb choose which editor to use.

I'm on Macbook. The first time I used blurb, it opened the default and unfamiliar editor, I almost didn't know how to exit 😓

Because I've been following core-workflow and devguide updates, I knew that the default editor can be changed by setting the EDITOR env variable. New users wouldn't know this and will have to look at the code to find out.